src/Knp/Menu/Renderer/ListRenderer.php

```diff
@@ -130,7 +131,11 @@ protected function renderItem(ItemInterface $item, array $options): string
         if ($this->matcher->isCurrent($item)) {
             $class[] = $options['currentClass'];
         } elseif ($this->matcher->isAncestor($item, $options['matchingDepth'])) {
-            $class[] = $options['ancestorClass'];
+            if (isset($options['ancestorClass']) { // Deprecated: it will be removed in future (@see: ancestor_class)
+                trigger_deprecation('knplabs/knp-menu', '3.2', 'Using "%s" option is deprecated, use "%s" instead.', 'ancestorClass', 'ancestor_class');
```

Comment:
We'll publish this in 3.3, not in 3.2

Reply:
Done in https://github.com/KnpLabs/KnpMenu/pull/351/commits/3aeb6874dbdb0fc156a4af600d35245c51907ddb